src/components/display.js

```diff
@@ -0,0 +1,10 @@
+import './display.css';
+
+function Display() {
+  return (
+    <p className="display">0</p>
+  );
+}
+// const Display = () => { <p className="display">0</p>; };
```

Comment:
- [ ] As I mentioned above make your code clean, kindly remove commented codes from your code!

Reply:
Thanks a lot, I'll do that 😄